### PR TITLE
[ACR-1695] Add the JPEG-XL description

### PR DESCRIPTION
**Pull request by jiyong2**
### API Changes ###

In case of ACR, put the link of ACR

For instance,

 - ACR: ACR-1695

### CODE REVIEW

**Review by omar-srbd — APPROVED** [review]
As the changes made to the file in this PR are minimal and everything is in order after a thorough check.

All the changes on other parts of the file in this PR have been tracked and will be incorporated through a new PR at a later time.

@asm-foysal @mijin85cho I am approving this PR for merging.

File: docs/application/native/guides/multimedia/image-edit.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ The main features of the Image Util API include:
   You can [decode images](#decode), [encode them](#encode), and [encode animation](#animation) with the following formats:
 
   - Bitmap formats: YUV420, YUV422, RGB888, RGBA8888, BGRA8888, and ARGB8888
-  - Input image formats for decoding: JPEG, PNG, GIF, BMP, and WEBP
-  - Output image formats for encoding: JPEG, PNG, GIF, BMP, and WEBP
+  - Input image formats for decoding: JPEG, PNG, GIF, BMP, WEBP, HEIF, and JPEG-XL
+  - Output image formats for encoding: JPEG, PNG, GIF, BMP, WEBP, and JPEG-XL
```
Comment (omar-srbd) [inline docs/application/native/guides/multimedia/image-edit.md:30]: Global
@jiyong2,
Please let me know if "**JPEG-XL**" be written as "**JPEG XL**"?
During a search on the internet, I found it to be written as "**JPEG XL**"
Ref: https://jpeg.org/jpegxl/

Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md:30]: @omar-srbd 
Oh, it's my mistake. I thought that it was a compound noun.
You are right, "JPEG XL" is a original.
I will replace the word to "JPEG XL". Thank you~!

Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md]: I updated it.

**Review by omar-srbd — COMMENTED** [review]
@jiyong2, thanks for incorporating the change.
After another review, I found a change that needs to be made.
Please refer to the suggestion and incorporate the change accordingly.
Thank you.

File: docs/application/native/guides/multimedia/image-edit.md [hunk 2]
```diff
@@ -544,25 +542,25 @@ The following tables define the supported color space formats.
 
 | Label                                    | FOURCC in hex | Bits per pixel      | Description                              |
 |------------------------------------------|---------------|---------------------|------------------------------------------|
-| [RGB](http://www.fourcc.org/rgb.php#BI_RGB) | 0x32424752    | 1, 4, 8, 16, 24, 32 | Alias for BI_RGB                         |
-| [RGBA](http://www.fourcc.org/rgb.php#RGBA) | 0x41424752    | 16, 32              | Raw RGB with alpha. Sample precision and packing is arbitrary and determined using bit masks for each component, as for BI_BITFIELDS. |
+| RGB | 0x32424752    | 1, 4, 8, 16, 24, 32 | Alias for BI_RGB                         |
```
[the remark this answers is not in the document]
Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md]: @omar-srbd, thanks for your suggestion. I added the missing period.

[the remark this answers is not in the document]
Comment (jiyong2) [reply docs/application/native/guides/multimedia/image-edit.md]: I added the missing period.

**Pull request merged**